### ISSUE TITLE
Voice prompts during navigation

### ISSUE THREAD

**Issue by GuentherGredy** [osmandapp/OsmAnd#17497]
# 🚀

**Comment by mikehgentry** [comment]
If you look in Navigation -> Settings -> Voice prompts (settings) -> Language -> Recorded

You'll find exactly that!

We hammered it out in #10501 so if you have feedback probably post it there.

Cheers!

**Comment by mikehgentry** [comment]
I think you raise a valid point. Maybe "Voice prompts" could be changed to "Audio prompts", and "Language" could arguably also be changed to "Type / Language" (since that better reflects the choice between TTS and recorded voice anyway).

It also wouldn't do any harm in my opinion mentioning the beep options in the documentation...

https://osmand.net/docs/user/navigation/guidance/voice-navigation/

I suppose I'll try to write a PR if I can figure out how to do that

**Comment by GuentherGredy** [comment]
If you change "Voice prompts" to "Audio prompts" then you could add and entry "Beeps" or something like that above "Type / Language". Both then get a switch control to enable / disable it (mutually excluded).
What is actually set could be shown in smaller letters beneath the entry. This kind of UI would ideally fit to many of that kind in the application.
Please, as an idea, refer to attached picture quickly drawn in MS Visio.
Thanks for you attention.
BR Guenther
![Suggestion](https://github.com/osmandapp/OsmAnd/assets/85982948/a7de8cc3-631e-4e11-8c9c-da4f1bda3ade)

**Comment by sonora** [comment]
I tend to agree that the user experience could likely benefit from restructuring the menu hierarchy. 

Today the user finds Navigation settings > Voice prompts > TTS/Recorded > Language

The beeps are not easiliy found. And also I have a feeling too many users still select "Recorded" over the preferable TTS, perhaps because TTS simply sounds unfamiliar. So if restructuring plus recording somehow mitigates these two issues, it may be worth it. Having said that, I believe that the vast majority of users should simply get (perhaps preconfigured) their device's system language as arTTS, so that option should not be buried deep in the hierarchy.

Perhaps something like Navigation settings > Audio guidance > languages (which would mean TTS), and there we would offer Beeps and (fallback) Recorded voices under "further options"?

**Comment by GuentherGredy** [comment]
Thank you for your message. Yes, I also stumbled across the abbreviation TTS and had to google it first :=); maybe this comes from the IT industry and is not familiar to everyone. Otherwise, I like your suggestion not to make the menu too deep and to restructure it the way you suggested. I look forward to the update here. Many thanks and best regards Guenther

**Comment by ossilator** [comment]
i think the new title of this issue is highly misleading. it should be more like "restructure configuration of audio navigation".
